packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -2103,18 +2103,7 @@ function invokeLayoutEffectUnmountInDEV(fiber: Fiber): void {
       case ClassComponent: {
         const instance = fiber.stateNode;
         if (typeof instance.componentWillUnmount === 'function') {
-          invokeGuardedCallback(
-            null,
-            safelyCallComponentWillUnmount,
-            null,
-            fiber,
-            instance,
-            fiber.return,
-          );
-          if (hasCaughtError()) {
-            const unmountError = clearCaughtError();
-            captureCommitPhaseError(fiber, fiber.return, unmountError);
-          }
+          safelyCallComponentWillUnmount(fiber, instance, fiber.return);
```

Comment:
We didn't need to wrap `safelyCallComponentWillUnmount` with `invokeGuardedCallback`, because it already uses `invokeGuardedCallback` internally.